README.md

```diff
@@ -22,13 +20,68 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### Ingestion
+
+Build client:
+
+```sh
+  client = Ubersicht::Ingestion::Client.new(
+    hmac_key: '44782DEF547AAA06C910C43932B1EB0C71FC68D9D0C057550C48EC2ACF6BA056',
+    account_id: '1001',
+    pass: 'password',
+    provider: 'DAuth',
+    url: '<api-base-url>',
+    user: 'user'
+  )
+```
+
+Parameters:
+
+* `hmac_key` - used for signing notification on client side and validation on server side (Ubersicht plugin config)
+* `account_id` - account which receives notification events
+* `pass` - basic auth password (Ubersicht plugin config)
+* `user` - basic auth username (Ubersicht plugin config)
+* `provider` - plugin provider in Ubersicht platform, e.g. DAuth
+* `url` - Ubersicht API root url, is different testing an production environment
+
+Send events:
+
+```sh
+  events = [
+    {
+      event_code: 'REQUESTED',
+      event_date: 2021-10-10 10:10:10,
+      transaction_id, 'eb2bc8bb-f584-4801-b98c-361a0c2d38f8',
+      type: 'DeviceBinding'
+      payload, {
+        device_id: '500',
+        user_id: '1000'
+      }
+    }
+  ]
+  client.ingest_events([events])
```

Comment:
pls remove array from here because now you have `[[{...}]]`

Reply:
Oh, thanks!